### PR TITLE
guide: update Node version (docs contrib)

### PR DESCRIPTION
**Pull request by jorgeorpinel**
`yarn` started failing for me today. It wanted node 14+ I think (I closed the terminal, oops).

### CODE REVIEW

**Comment by rogermparent** [comment]
Our `package.json` is set for `"node": "<=16.x"`, 16 being the current LTS and 14 being the last one.

**Comment by jorgeorpinel** [comment]
I had 12 (the default from `apt-get` on Ubuntu) which worked up til yesterday for me.

File: content/docs/user-guide/contributing/docs.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ the website engine itself. Source code and content files need to be properly
 formatted and linted as well, which is also ensured by the full setup below.
 
 Make sure you have [Python](https://www.python.org/downloads/) 3.7+, a recent
-LTS version of [Node.js](https://nodejs.org/en/) (`>=12.0.0`, `<=15.x`), and
+LTS version of [Node.js](https://nodejs.org/en/) (`>=14.0.0`, `<=17.x`), and
```
Comment (jorgeorpinel) [inline content/docs/user-guide/contributing/docs.md:60]: So is this the correct change then @rogermparent ?
```suggestion
LTS version of [Node.js](https://nodejs.org/en/) (`>=14.0.0`, `<=16.x`), and
```

Comment (jorgeorpinel) [reply content/docs/user-guide/contributing/docs.md:60]: Yeah I guess this should be it. Committing, please approve/merge.

**Review by rogermparent — APPROVED** [review]

**Comment by rogermparent** [comment]
We should also update the `engines.node` field in `package.json`. According to [yarn docs](https://classic.yarnpkg.com/en/docs/package-json#toc-engines), our value should be `">=14.x <=16.x"`.

**Comment by skshetry** [comment]
> We should also update the `engines.node` field in `package.json`. According to [yarn docs](https://classic.yarnpkg.com/en/docs/package-json#toc-engines), our value should be `">=14.x <=16.x"`.

I just tried yesterday with node 17 and node 16. Both failed while installing/compiling `sharp`.

**Comment by jorgeorpinel** [comment]
> We should also update the engines.node field in package.json

Done @rogermparent . I removed node_modules and ran `yarn` again successfully. PTAL

> tried yesterday with node 17 and node 16. Both failed

@skshetry does that relate to `engines.node`? Can you provide more context? I'm on WSL2/Ubuntu with Node 16 and this currently works fine.

**Comment by rogermparent** [comment]
> I just tried yesterday with node 17 and node 16. Both failed while installing/compiling `sharp`.

Works for me on node v16.11.1, 17 should be rejected by yarn with the message "Found incompatible module." (though it also works for me if forced by changing the `engines` versions).

As far as this PR's changes, they look fine to me.

**Review by rogermparent — APPROVED** [review]

**Pull request merged**